src/app/auth/views/Auth/signup.ts

```diff
@@ -10,7 +10,7 @@ const postMessage = (data: Record<string, unknown>) => {
   window.top?.postMessage(data, CampaignLinks.PcComponentes);
 };
 
-const signup = async (data, dispatch, doRegister, setLoading, setError?) => {
+const signup = async (data, dispatch, doRegister?, setLoading?, setError?) => {
```

Comment:
Why are doRegister and setLoading optional, if their value is undefined won't it crash?